newIDE/app/src/AssetStore/ExampleStore/ExampleDialog.js

```diff
@@ -173,10 +173,10 @@ export function ExampleDialog({
         </ResponsiveLineStackLayout>
 
         {example && example.description && (
-          <>
+          <Column>
             <Divider />
             <MarkdownText source={example.description} isStandaloneText />
```

Comment:
Th markdown text here and the text above do not have the same font size, it's a bit unsettling but ok

Reply:
fixed